.travis.yml

```diff
@@ -3,12 +3,15 @@ rust:
   - stable
   - nightly
 cache: cargo
-env:
-- MCU=stm32f042
 matrix:
   allow_failures:
     - rust: nightly
   fast_finish: true
 script:
   - rustup target add thumbv6m-none-eabi
-  - cargo build --features=$MCU
+  - cargo build --features=stm32f042
```

Comment:
You don't want to do that. Please keep as is and add the features to the `env` block.